pywbem/cim_operations.py

```diff
@@ -636,6 +658,26 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_operation_time = None
         self._last_server_response_time = None
 
+        # logging
+        self.enable_logging = enable_logging
```

Comment:
Instead of adding another undocumented public attribute, I suggest that we add a documented read-write property for "enabled_logging".

Reply:
**DISCUSSION.** This means that we are moving the enable of logging from just part of the init so something that can be executed at any point after the connection is created.

There is one inconsistency in that for stats it is stats_enabled and we would have enabled_logging.

Also we have to consider the fact that we log the execution of the connection itself. If this were set after we  create the object, at the very least, the time in the log for the connection would be the time the property was set, not the time the connection was created.